packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts

```diff
@@ -29,6 +29,26 @@ import {
   Version,
 } from './version';
 
+/**
+ * Provider for adding user data scripts
+ */
+export interface IInstanceUserDataProvider {
```

Comment:
Do we want another callback to run before the worker launch? I can imagine a case where someone might want to set something up after Deadline is already configured to properly connect to the Render Queue, but before the Worker is launched.